src/components/simpleInput/Input.js

```diff
@@ -0,0 +1,80 @@
+import './input.scss';
+
+class Input extends Component {
+  state = {
+    inputText: '',
+    callback: null
+  };
+
+  changeField = (event) => {
+    this.setState({ inputText: event.target.value });
+    this.props.setInputText(this.state.inputText);
```

Comment:
1) why do you save input text in the state here and in the state of `SimpleInput` component?
2) setState is async function, so you pass to `this.props.setInputText` an obsolete value, it doesn't work as expected

Have you checked how your component works? Check this short [video](https://www.dropbox.com/s/4mf37fov1uapxnv/inputIssue.gif?dl=0) to understand what I'm talking about